modules/claims_api/lib/bgs_service/vdc_manage_representative_service.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class VdcManageRepresentativeService < ClaimsApi::LocalBGS
+    def bean_name
+      'VDC/ManageRepresentativeService'
+    end
+
+    def update_poa(rep, proc_id)
+      body = get_update_poa_body(rep, proc_id)
+
+      make_request(
+        endpoint: bean_name,
+        action: 'updatePOARequest',
+        body:,
+        additional_namespace:
+      )
+    end
+
+    private
+
+    def additional_namespace
+      Nokogiri::XML::DocumentFragment.parse <<~EOXML
+        xmlns:data='http://gov.va.vba.benefits.vdc/data'
```

Comment:
there are quite a few variations of this in vets-api, and maybe it's _not_ concerning but I don't like having the full path here. 🤔 trying to think of alternate solutions though.

Reply:
@FonzMP   I made som adjustments to that.  The testing still worked as expected, check it out and let me know if that is more in line with what you would like to see.  At this stage I know it will work with mine, the question would be will it work with all, but I can only find that out as more additional namespaces are utilized.